internal/pkg/deploy/cloudformation/stack/worker_svc.go

```diff
@@ -0,0 +1,157 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package stack
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/addon"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+	"github.com/aws/copilot-cli/internal/pkg/template"
+)
+
+type workerSvcReadParser interface {
+	template.ReadParser
+	ParseWorkerService(template.WorkloadOpts) (*template.Content, error)
+}
+
+// WorkerService represents the configuration needed to create a CloudFormation stack from a worker service manifest.
+type WorkerService struct {
+	*ecsWkld
+	manifest *manifest.WorkerService
+
+	allowedTopics []string
+	parser        workerSvcReadParser
+}
+
+// NewWorkerService creates a new WorkerService stack from a manifest file.
+func NewWorkerService(mft *manifest.WorkerService, env, app string, rc RuntimeConfig, at []string) (*WorkerService, error) {
```

Comment:
```suggestion
func NewWorkerService(mft *manifest.WorkerService, env, app string, rc RuntimeConfig, allowedTopics []string) (*WorkerService, error) {
```

Can we make this input name more descriptive? Will help readability in other packages.